trunk/source/Library-ModelCheckerUtils/src/de/uni_freiburg/informatik/ultimate/lib/modelcheckerutils/cfg/IcfgUtils.java

```diff
@@ -221,6 +221,28 @@ public static <LOC extends IcfgLocation> boolean areRegisteredProgramPointsReach
 		return true;
 	}
 
+	public static <LOC extends IcfgLocation> boolean areLocationsOfInterestRegistered(final IIcfg<LOC> icfg) {
```

Comment:
These two methods are almost identical.
Also you just check if the set difference is empty. There should be already a method in `DataStructureUtils` to do this that can be used here for simplification.

Reply:
https://github.com/ultimate-pa/ultimate/pull/713/commits/df8f5a7ea5970d153c4d407d597d9ec77706a769